### DISCUSSION TITLE
New release, 2.15

### DISCUSSION THREAD

**Discussion by Sp3EdeR** [luvit/luvi#293] [General]
Hello maintainers,
I would like to ask whether there is a plan to release v2.15. There have been some fixes and updates to master, and a long time has passed since 2.14. It would be nice if we could have a new release.

**Comment by SinisterRectus** [comment]
I believe we are waiting for a consensus on how to handle (or not handle) the remaining platforms in #200.

**Comment by Bilal2453** [comment]
The remaining platforms, as [Zhaozg mentioned](https://github.com/luvit/luvi/issues/200), will likely be a long-time task due to toolchain challenges, as well as lower priorities.
Probably not worth waiting for them.

**Comment by truemedian** [comment]
Nobody is actively working towards supporting them, and we're not dropping support for any currently supported package in this update. I see no problem with slating out 2.15, any other platforms can be introduced in 2.16+.

**Comment by truemedian** [comment]
Release has been published: https://github.com/luvit/luvi/releases/tag/v2.15.0
